app/models/form/subsection.rb

```diff
@@ -58,7 +58,7 @@ def applicable_questions(case_log)
   end
 
   def answered_questions(case_log)
-    applicable_questions(case_log).select { |question| case_log[question.id].present? }
+    applicable_questions(case_log).select { |question| question.type == "checkbox" ? question.answer_options.keys.any? { |key| case_log[key] == "Yes" } : case_log[question.id].present? }
```

Comment:
This could probably also be switched to using `question.completed?`?

Reply:
Will update this after updating the completed method, especially since there are tests now 🙂